app/src/main/java/org/wikipedia/readinglist/ReadingListItemView.kt

```diff
@@ -182,8 +196,76 @@ class ReadingListItemView : ConstraintLayout {
                     list?.let { callback?.onRemoveAllOffline(it) }
                     return true
                 }
+                R.id.menu_reading_list_export -> {
+                    AlertDialog.Builder(context)
```

Comment:
I'm not sure we need to show a dialog box before exporting the list. We can just export it.